### DISCUSSION TITLE
smooth_add, smart_crossfade is not working ubunt 21.04

### DISCUSSION THREAD

**Discussion by xamke** [savonet/liquidsoap#1911] [Installation]
Hi, 

After install ubunt 21.04, some of functions not working with error.(liquidsoap 1.4.2 installed by apt-get install liquidsoap)

**Error 4: Undefined variable smooth_add**

even smart_corssfade show same error.

Please let me how to fix it.

Thank you

**Comment by smimram** [comment]
It is strange. Those functions are [still present in Liquidsoap 1.4.2](https://www.liquidsoap.info/doc-1.4.2/reference.html#smooth_add). I guess that the problem comes from the fact that they are defined in the standard library, which is not loaded for some reason?
